content/en/registry/instrumentation-js-contrib-nestjs.md

```diff
@@ -0,0 +1,13 @@
+---
+title: NestJS OpenTelemetry
+registryType: instrumentation
+isThirdParty: true
+language: js
+tags:
+  - Node.js
+	- instrumentation
+	- nestjs
```

Comment:
Replace the tabs by spaces in these two lines too.

Reply:
should be fix